ingests_tracker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApiFeatureTest.scala

```diff
@@ -19,12 +23,54 @@ class IngestsTrackerApiFeatureTest
     with IntegrationPatience
     with StorageRandomThings {
 
-  it("starts") {
-    withConfiguredApp() { _ =>
-      val path = s"http://localhost:8080/ingests/id"
+  describe("GET /healthcheck") {
+    it("responds OK") {
+      withConfiguredApp() { _ =>
+        val path = s"http://localhost:8080/healthcheck"
+
+        whenGetRequestReady(path) { result =>
+          result.status shouldBe StatusCodes.OK
+        }
+      }
+    }
+  }
+
+  describe("GET /ingest/:id") {
+
+    val ingest = createIngestWith(
+      createdDate = Instant.now(),
+      events = Seq(createIngestEvent, createIngestEvent).sortBy {
+        _.createdDate
+      },
+      version = None
+    )
+
+    val notRecordedIngestId = createIngestID
+
+    it("responds NotFound when there is no ingest") {
+      withConfiguredApp(Seq(ingest)) { _ =>
+        val path = s"http://localhost:8080/ingest/${notRecordedIngestId}"
+
+        whenGetRequestReady(path) { result =>
+          result.status shouldBe StatusCodes.NotFound
+        }
+      }
+    }
+
+    it("responds OK with an Ingest when available") {
+      withConfiguredApp(Seq(ingest)) { _ =>
+        val path = s"http://localhost:8080/ingest/${ingest.id}"
+
+        whenGetRequestReady(path) { result =>
+          result.status shouldBe StatusCodes.OK
 
-      whenGetRequestReady(path) { result =>
-        result.status shouldBe StatusCodes.NotFound
+          // We are interested in whether we can serialise/de-serialise
+          // what the JSON looks like is not interesting, so we
+          // do not need to test in detail as with the external API
+          withStringEntity(result.entity) { jsonString =>
+            fromJson[Ingest](jsonString).get shouldBe ingest
+          }
+        }
       }
     }
```

Comment:
Also needs a test for the HTTP 500 endpoint.

Reply:
👍 is in next PR.